google/resource_sql_database_instance_test.go

```diff
@@ -9,7 +9,6 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
-
```

Comment:
@slevenick  Just trying to understand how this change generates inspec for beta features?

Reply:
This is unrelated and caused by some difference in the go format tool. This PR will be closed.